### PR TITLE
Rest for config and scan. Discovery-config

### PR DESCRIPTION
**Pull request by yarovyii**

### CODE REVIEW

File: common/commands.cpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            switch (value) {
+                case Status::CancelledByUser:
+                    return "1";
+                case Status::Terminated:
+                    return "2";
+                case Status::InProgress:
+                    return "3";
+                case Status::Unknown:
+                    return "0";
+            }
```
Comment (etnKlendathu) [inline common/commands.cpp:64]: at least 
```
ss << fty::convert<int>(value);
```
:)
But I don't understand why in this special case you need numeric values :(

File: common/commands.h [hunk 2]
```diff
@@ -111,4 +111,133 @@ namespace commands::assets {
...
+
+    } // namespace status
+
+    struct Response : public pack::Node
+    {
+        enum class Status
+        {
+            Success,
+            Fail,
+            Unknown
```
Comment (etnKlendathu) [inline common/commands.h:184]: Move Unknown to top everywhere.

File: rest/CMakeLists.txt [hunk 3]
```diff
@@ -2,14 +2,24 @@
 
 etn_target(shared ${PROJECT_NAME}-rest
     SOURCES
-        conf/42_discovery.xml.in
```
Comment (etnKlendathu) [inline rest/CMakeLists.txt:5]: why did you remove this? This file now is unhandled by cmake at all

**Review by etnKlendathu — CHANGES_REQUESTED** [review]

File: rest/CMakeLists.txt (hunk 3 above)
Comment (yarovyii) [reply rest/CMakeLists.txt:5]: in fty-automatic-group implemented in same way
